docs/docs/writing-stories.mdx

```diff
@@ -105,6 +105,7 @@ stories:
    - action: action_check_profile
    - slot_was_set:
      - premium_account: true
+     - account_data: '{"id":"zpNi3JA2", "fullName":"Ada Lovelace", "accountRenewalDate":"October 3, 2021"}'
```

Comment:
Same for this - a slot of type any should not be included in a story, as it won't make any difference